CHANGELOG.md

```diff
@@ -31,7 +31,9 @@
 * Connection in bytes_connection.go renamed to BytesConnection
 * kafka plugin initializes two multiplexers for dynamic mode (automatic partitions) and manual mode.
   Every multiplexer can create its own connection and provides access to different set of methods 
-  (publishing to partition, watching on partition/offset) 
+  (publishing to partition, watching on partition/offset)
+* ProtoWatcher from API was split to ProtoWatcher for dynamic mode and ProtoManualWatcher to listen on
```

Comment:
please wordsmith (it was not split - since the original ProtoWatcher still exists...)

please mention that the API now supports manual marking of offsets